labels.md

```diff
@@ -0,0 +1,219 @@
+component/docs                  #e99695     C: Docs                             #d1fad7
+component/document store        #e99695     C: Document store                   #d1fad7
+component/executor              #000000     C: Executor                         #d1fad7
+component/expression            #fbca04     C: Expression                       #d1fad7
+component/metrics               #e26f88     C: Metrics                          #d1fad7
+component/mysql-protocol        #66dde8     C: MySQL protocol                   #d1fad7
+component/parser                #fbca04     C: Parser                           #d1fad7
+component/planner               #1d76db     C: Planner                          #d1fad7
+component/plugin                #c98fe8     C: Plugin                           #d1fad7
+component/privilege             #81d5e2     C: Privilege                        #d1fad7
+component/server                #e0854c     C: Server                           #d1fad7
+component/session               #49309b     C: Session                          #d1fad7
+component/statistics            #80e58d     C: Statistics                       #d1fad7
+component/syncer                #ccbf0e     C: Syncer                           #d1fad7
+component/test                  #6494f4     C: Test                             #d1fad7
+component/tikv                  #8bef73     C: Tikv                             #d1fad7
+component/tools                 #cbf232     C: Tools                            #d1fad7
+component/transaction-need-LGT  #83e2d1     C: Transaction need LGT             #d1fad7
+component/usability             #0ad8bd     C: Usability                        #d1fad7
+component/util                  #47e0b0     C: Util                             #d1fad7
+contribution                    #b0ff5b     T: Contributor ⭐️                    #1d76db
+errmsg                          #a7c938     T: Error message                    #1d76db
+for new contributors            #c2e0c6     D: Mentor                           #0e8a16
+help wanted                     #159818     S: HelpWanted                       #e6e6e6
+needs-cherry-pick-2.1           #000000     T: CherryPick-2.1                   #1d76db
+needs-cherry-pick-3.0           #000000     T: CherryPick-3.0                   #1d76db
+priority/P1                     #b60205     P: Low                              #eb6420
+priority/P2                     #FF7F50     D: Medium                           #0e8a16
+priority/non-release-blocker    #0052cc     P: High                             #eb6420
+priority/release-blocker        #b60205     P: Blocker                          #eb6420
+proposal                        #eb6420     T: Proposal                         #1d76db
+question                        #fbca04     T: Question                         #1d76db
+release-note                    #1d76db     * <T: CHANGELOG>                    
```

Comment:
Maybe we should remove this label, seems it's duplicated with the `CHANGELOG` label.

Reply:
The `*` means it should be removed in favour of `T: CHANGELOG`